examples/modules/06a-rename-import/example.effekt

```diff
@@ -0,0 +1,5 @@
+import hello.world as hw
```

Comment:
Scala uses the following syntax for renaming imports:

```suggestion
import hello.{ world => hw }
```

An advantage is that this way you can import multiple names in one line. That is,

```
import foo.bar as fb
import foo.bam as fm
```
becomes
```
import foo. { bar => fb, bam => fm}
```

We could also use the keyword `as` instead of overloading the `=>` operator. So this looks like 
```
import foo. { bar as fb, bam as fm}
```